src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -222,4 +228,29 @@ describe('Reimbursement Requests', () => {
       expect(prisma.reimbursement_Request.update).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Delivered Tests', () => {
+    test('Mark as delivered fails for non admins', async () => {
+      await expect(
+        ReimbursementRequestService.markReimbursementRequestAsDelivered(wonderwoman, PopEyes.vendorId)
+      ).rejects.toThrow(new AccessDeniedException('Only the creator of the reimbursement request can mark as delivered'));
+    });
+
+    test('Mark as delivered fails for undefined ID', async () => {
+      await expect(ReimbursementRequestService.markReimbursementRequestAsDelivered(batman, '')).rejects.toThrow(
+        new NotFoundException('Reimbursement Request', requestDeliveredValid.reimbursementRequestId)
+      );
+    });
+
+    test('Mark request as delivered successfully', async () => {
+      jest.spyOn(prisma.reimbursement_Request, 'findUnique').mockResolvedValue(requestDeliveredValid);
```

Comment:
from the comment above, you would copy and paste this line and instead of requestDeliveredValid, you would just say null